apps/aecore/test/aec_chain_sim.erl

```diff
@@ -269,7 +291,7 @@ remove_meck() ->
 init(Opts) when is_map(Opts) ->
     gproc:reg({n,l,{?MODULE, chain_process}}),
     Chain = new_chain(),
-    ?LOG("Initial chain: ~p", [Chain]),
+    ?LOG("Initial chain (~p simulator): ~p", [simulator(Opts), Chain]),
```

Comment:
Log all opts otherwise this will hard crash in case someone passes in wrong opts.

Reply:
Agree